controllers/modelmesh/modelmesh.go

```diff
@@ -281,6 +314,9 @@ func (m *Deployment) addMMEnvVars(deployment *appsv1.Deployment) error {
 }
 
 func (m *Deployment) setConfigMap() error {
+	if m.Owner == nil {
+		return nil
+	}
```

Comment:
This can use annotations stored in the Deployment struct (see other comment about that).